lib/plug/adapters/test/conn.ex

```diff
@@ -22,6 +22,11 @@ defmodule Plug.Adapters.Test.Conn do
       chunks: nil,
       ref: make_ref(),
       owner: owner,
+      peer_data:
+        case conn.adapter do
+          {Plug.MissingAdapter, _} -> %{address: {127, 0, 0, 1}, port: 111_317, ssl_cert: nil}
+          {adapter, payload} -> adapter.get_peer_data(payload)
+        end,
```

Comment:
Let's encapsulate this in a function given this and the code below do the same thing?

For example, `get_from_adapter(conn, op, default)` which we would call as `get_from_adapter(conn, :get_peer_data,  %{address: {127, 0, 0, 1}, port: 111_317, ssl_cert: nil})`?